data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -16,6 +16,7 @@
 import static tech.pegasys.teku.ethereum.json.types.EthereumTypes.SIGNATURE_TYPE;
 import static tech.pegasys.teku.infrastructure.http.RestApiConstants.ATTESTATION_DATA_ROOT;
 import static tech.pegasys.teku.infrastructure.http.RestApiConstants.BEACON_BLOCK_ROOT;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.BLOCK_ROOT;
```

Comment:
Not sure whether to simply reuse `BEACON_BLOCK_ROOT`, they are equivalent data.

Reply:
yep if it's the same exact structure, just re-use